pkg/worker/options.go

```diff
@@ -33,9 +33,12 @@ type WorkerOptions struct {
 	// ca cert file for vault api client, if vault used a self signed certificate
 	CaCertFile string
 
-	// If InSecureTLS true, then it will skip tls verification when communicating with vault server
+	// If InSecureTLS is true, then it will skip tls verification when communicating with vault server
 	InSecureTLS bool
 
+	// If SkipReadWriteAccessCheck is true, then it will skip read/write access check before initializing the vault
+	SkipReadWriteAccessCheck bool
```

Comment:
Do we need this flag?

Reply:
I keep this flag, if user don't want to give delete permission to unsealer